examples/experimental/trial/unets_tf_keras/startup-hook.sh

```diff
@@ -0,0 +1,3 @@
+pip install git+https://github.com/tensorflow/examples.git
+pip install -q -U tfds-nightly
```

Comment:
Are you still using tfds-nightly and the register_checksums since you are downloading the data file manually?

Reply:
The data is actually downloaded in the startup hook with the checksum. The only the model code is downloaded during training.